src/cascadia/TerminalSettingsEditor/GlobalSettingsViewModel.idl

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "EnumEntry.idl";
+
+#include "ViewModelHelpers.idl.h"
+
+namespace Microsoft.Terminal.Settings.Editor
+{
+    runtimeclass GlobalSettingsViewModel : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    {
+        // REGION: Launch

```

Comment:
I should get rid of this `// REGION` stuff and organize it like this:
- view model state properties & misc
- pairings of `CurrentXXX` and `XXXList`
- projected settings